docs/clusterfuzz/architecture.md

```diff
@@ -21,35 +22,41 @@ triaging crashes, minimizing reproducers, [bisecting], and verification of fixes
 ClusterFuzz is written in Python and Go. It runs on **Linux**, **macOS**, and **Windows**.
 
 ## Requirements
-It runs on the [Google Cloud Platform](https://cloud.google.com/), and depends
+ClusterFuzz runs on the [Google Cloud Platform](https://cloud.google.com/) and depends
 on a number of services:
-- Compute Engine (Not strictly necessary. Bots can run anywhere).
+- Compute Engine (Not strictly necessary. [Fuzzing bots](#bots) can run anywhere.)
 - App Engine
 - Cloud Storage
 - Cloud Datastore
 - Cloud Pub/Sub
 - BigQuery
 - Stackdriver Logging and Monitoring
 
-**Note**: The only bug tracker supported now is the Chromium hosted
+**Note**: The only bug tracker we currently support is the Chromium-hosted
 [Monorail](https://opensource.google.com/projects/monorail). Support for custom
-bug trackers will be added in the near future.
+bug trackers will be added in the future.
 
 ### Local instance
 It's possible to run ClusterFuzz locally without these dependencies by using
-local Google Cloud emulators, but some features which depend on BigQuery and
+local Google Cloud emulators. If you do, some features that depend on BigQuery and
 Stackdriver will be disabled due to lack of emulator support.
 
+**Note:** Local development is only supported on **Linux** and **macOS**.
```

Comment:
maybe "instances are" instead of "development is"

Reply:
Done.